### PR TITLE
Update libgcrypt-version to 1.11.0 from 1.10.3

### PR DESCRIPTION
**Pull request by zosopentoolsmain**

### CODE REVIEW

**Comment by HarithaIBM** [comment]
Java error on jenkins system. rerunning build:

Building remotely on zOS Cloud Instance 6 - 163.74.95.209 (go_120 clang v2r4 zos) in workspace /jenkins/workspace/Port-Build
FATAL: java.io.IOException: Unexpected termination of the channel
java.io.EOFException
	at java.base/java.io.ObjectInputStream$PeekInputStream.readFully(ObjectInputStream.java:2915)
	at java.base/java.io.ObjectInputStream$BlockDataInputStream.readShort(ObjectInputStream.java:3410)
	at java.base/java.io.ObjectInputStream.readStreamHeader(ObjectInputStream.java:954)
	at java.base/java.io.ObjectInputStream.<init>(ObjectInputStream.java:392)
	at hudson.remoting.ObjectInputStreamEx.<init>(ObjectInputStreamEx.java:50)
	at hudson.remoting.Command.readFrom(Command.java:141)
	at hudson.remoting.Command.readFrom(Command.java:127)
	at hudson.remoting.AbstractSynchronousByteArrayCommandTransport.read(AbstractSynchronousByteArrayCommandTransport.java:35)
	at hudson.remoting.SynchronousCommandTransport$ReaderThread.run(SynchronousCommandTransport.java:62)
Caused: java.io.IOException: Unexpected termination of the channel
	at hudson.remoting.SynchronousCommandTransport$ReaderThread.run(SynchronousCommandTransport.java:80)
Also:   hudson.remoting.Channel$CallSiteStackTrace: Remote call to zOS Cloud Instance 6 - 163.74.95.209
		at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1923)
		at hudson.remoting.Request.call(Request.java:204)
		at hudson.remoting.Channel.call(Channel.java:1111)
		at hudson.FilePath.act(FilePath.java:1228)
		at hudson.FilePath.act(FilePath.java:1217)
		at hudson.FilePath.mkdirs(FilePath.java:1408)
		at hudson.model.AbstractProject.checkout(AbstractProject.java:1244)
		at hudson.model.AbstractBuild$AbstractBuildExecution.defaultCheckout(AbstractBuild.java:649)
		at jenkins.scm.SCMCheckoutStrategy.checkout(SCMCheckoutStrategy.java:85)
		at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:522)
		at hudson.model.Run.execute(Run.java:1[83](https://github.com/zopencommunity/libgcryptport/actions/runs/11857696429/job/33046657776#step:5:84)1)
		at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:44)
		at hudson.model.ResourceController.execute(ResourceController.java:101)
		at hudson.model.Executor.run(Executor.java:445)
Caused: hudson.remoting.RequestAbortedException
	at hudson.remoting.Request.abort(Request.java:358)
	at hudson.remoting.Channel.terminate(Channel.java:1196)
	at hudson.remoting.SynchronousCommandTransport$ReaderThread.run(SynchronousCommandTransport.java:95)
ERROR: Step ?Archive the artifacts? failed: no workspace for Port-Build #5580
ERROR: Step ?Delete workspace when build is done? failed: no workspace for Port-Build #5580
Finished: FAILURE
Job failed...

**Review by IgorTodorovskiIBM — APPROVED** [review]

File: buildenv [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # bump: libgcrypt-version /LIBGCRYPT_VERSION="(.*)"/ https://gnupg.org/ftp/gcrypt/libgcrypt/|re:/libgcrypt-([\d.]+).tar.bz2/$1/|semver:*
-LIBGCRYPT_VERSION="1.10.3"
+LIBGCRYPT_VERSION="1.11.0"
 
 export ZOPEN_STABLE_URL="https://gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-${LIBGCRYPT_VERSION}.tar.bz2"
 export ZOPEN_STABLE_DEPS="make bzip2 tar curl perl automake autoconf m4 sed gettext zoslib diffutils libgpgerror"
 export ZOPEN_BUILD_LINE="STABLE"
-export ZOPEN_EXTRA_CFLAGS="${ZOPEN_EXTRA_CFLAGS} -O3"
+export ZOPEN_EXTRA_CFLAGS="${ZOPEN_EXTRA_CFLAGS} -O0"
```
[the remark this answers is not in the document]
Comment (HarithaIBM) [reply buildenv:7]: Sure Igor, raised this one to track it, https://github.com/zopencommunity/libgcryptport/issues/12

**Pull request merged**